resolwe_bio/docker_images/dnaseq/packages-debian-buildreq.txt

```diff
@@ -1,10 +1,19 @@
 build-essential
 python3-dev
-libcurl4-openssl-dev
 zlib1g-dev
 libbz2-dev
 liblzma-dev
 
+# Required for R package 'devtools'
+libcurl4-openssl-dev
+libssl-dev
+
+# Required for R package 'ade4'
```

Comment:
Where is 'ade4' installed?

Reply:
It's one of the of the dependencies of the Bioconductor suite. I've modified the note to reflect this.